.github/workflows/build.yaml

```diff
@@ -0,0 +1,440 @@
+name: indy-node-build
+on: [ push, pull_request, workflow_dispatch ]
```

Comment:
@udosson, I think the following `on` section from `indy-plenum` should be used here:
https://github.com/hyperledger/indy-plenum/blob/4547fc48d113e854f49c38569a26d645d3f5608e/.github/workflows/build.yaml#L2-L11

Reply:
agreed!